javascript-loops-practice/includes-seven.js

```diff
@@ -0,0 +1,27 @@
+/* exported includesSeven */
+function includesSeven(array) {
+  var sevenIncluded = false;
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === 7) {
+      sevenIncluded = true;
+    }
+  }
+  if (sevenIncluded === true) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
With that last revision, you can now just `return sevenIncluded`, no conditional necessary here 🤓